src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -168,13 +168,12 @@ void goto_symext::symex_allocate(
 
   if(!zero_init.is_zero() && !zero_init.is_false())
   {
-    null_message_handlert null_message;
     exprt zero_value=
       zero_initializer(
         object_type,
         code.source_location(),
         ns,
-        null_message);
```

Comment:
This should use the throwing version, because really it actually throws below ...